content/en/docs/instrumentation/python/exporters.md

```diff
@@ -99,63 +246,90 @@ Setting `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to any other value than
 `CUMULATIVE`, `DELTA` or `LOWMEMORY` will log a warning and set this environment
 variable to `CUMULATIVE`.
 
-```python
-from opentelemetry import metrics
-from opentelemetry.sdk.metrics import MeterProvider
-from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader, ConsoleMetricExporter
-from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+{{% /alert %}}
 
-# Service name is required for most backends,
-# and although it's not necessary for console export,
-# it's good to set service name anyways.
-resource = Resource(attributes={
-    SERVICE_NAME: "your-service-name"
-})
+## Jaeger
 
-reader = PeriodicExportingMetricReader(ConsoleMetricExporter())
-provider = MeterProvider(resource=resource, metric_readers=[reader])
-metrics.set_meter_provider(provider)
+[Jaeger](https://www.jaegertracing.io/) natively supports OTLP to receive trace
+data. You can run Jaeger in a docker container with the UI accessible on port
+16686 and OTLP enabled on ports 4137 and 4138:
```

Comment:
I prefer port numbers _not_ to be in code font, as you have it here. But I wanted to note that this convention isn't used consistently in this page.